### ISSUE TITLE
Unspecified error [d7a9fb1643]

### ISSUE THREAD

**Issue by aem-intellij-plugin-error-reporter** [aem-intellij-plugin-error-reporter/auto-generated-issues-storage#37]
fixed:0.10.4

## Description
*No description*
## Exception
```
java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    dx.handleElementRename(Ljava/lang/String;)Lcom/intellij/psi/PsiElement; @34: invokestatic
  Reason:
    Type 'com/intellij/psi/PsiReference' (current frame, stack[1]) is not assignable to 'java/lang/String'
  Current Frame:
    bci: @34
    flags: { }
    locals: { 'dx', 'java/lang/String' }
    stack: { 'ed', 'com/intellij/psi/PsiReference', 'java/lang/String' }
  Bytecode:
    0x0000000: 2b12 07b8 0028 b800 1b59 c600 1f2a c000
    0x0000010: 0f2a b600 1e59 c700 06b8 0029 5912 0ab8
    0x0000020: 0027 b800 2259 c700 0557 2b59 4dc0 0015
    0x0000030: 594e b900 2a01 009e 0007 04a7 0004 0399
    0x0000040: 000f 2a2c b600 2059 1209 b800 27b0 2ac0
    0x0000050: 000d b0                                
  Stackmap Table:
    full_frame(@28,{Object[#19],Object[#23]},{Object[#20],Object[#15],Object[#23]})
    same_locals_1_stack_item_frame(@41,Object[#22])
    full_frame(@43,{Object[#19]},{Object[#23]})
    append_frame(@62,Top,Object[#23])
    same_locals_1_stack_item_frame(@63,Integer)
    chop_frame(@78,2)

	at cw$a.a(Unknown Source)
	at bT.createElement(Unknown Source)
	at com.intellij.psi.impl.source.tree.CompositeElement.createPsiNoLock(CompositeElement.java:739)
	at com.intellij.psi.impl.source.tree.CompositeElement.getPsi(CompositeElement.java:726)
	at com.intellij.psi.impl.source.SourceTreeToPsiMap.treeElementToPsi(SourceTreeToPsiMap.java:30)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.getNextSibling(SharedImplUtil.java:60)
	at com.intellij.extapi.psi.ASTDelegatePsiElement.getNextSibling(ASTDelegatePsiElement.java:110)
	at com.intellij.psi.SyntaxTraverser$PsiApi.next(SyntaxTraverser.java:308)
	at com.intellij.psi.SyntaxTraverser$PsiApi.next(SyntaxTraverser.java:264)
	at com.intellij.psi.SyntaxTraverser$ApiEx$1.fun(SyntaxTraverser.java:254)
	at com.intellij.util.containers.JBIterable$2$1.nextImpl(JBIterable.java:125)
	at com.intellij.util.containers.JBIterator.peekNext(JBIterator.java:150)
	at com.intellij.util.containers.JBIterator.hasNext(JBIterator.java:110)
	at com.intellij.util.containers.TreeTraversal$BiOrderIt.nextImpl(TreeTraversal.java:454)
	at com.intellij.util.containers.JBIterator.peekNext(JBIterator.java:150)
	at com.intellij.util.containers.JBIterator.hasNext(JBIterator.java:110)
	at com.intellij.codeInsight.daemon.impl.ChameleonSyntaxHighlightingPass.collectInformationWithProgress(ChameleonSyntaxHighlightingPass.java:91)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:69)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:430)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1166)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:423)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:586)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:86)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:422)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:398)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:218)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:396)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:174)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```
## Error context
```
Last Action:       null
Plugin Version:    0.10.3
OS Name:           Windows 10
Java Version:      1.8.0_202-release
Java VM Vendor:    JetBrains s.r.o
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IU-191.7141.44
App Version:       2019.1.2
```

**Comment by aem-intellij-plugin-error-reporter** [comment]
## Description
*No description*
## Exception
```
java.lang.VerifyError: Bad type on operand stack
Exception Details:
  Location:
    dx.handleElementRename(Ljava/lang/String;)Lcom/intellij/psi/PsiElement; @34: invokestatic
  Reason:
    Type 'com/intellij/psi/PsiReference' (current frame, stack[1]) is not assignable to 'java/lang/String'
  Current Frame:
    bci: @34
    flags: { }
    locals: { 'dx', 'java/lang/String' }
    stack: { 'ed', 'com/intellij/psi/PsiReference', 'java/lang/String' }
  Bytecode:
    0x0000000: 2b12 07b8 0028 b800 1b59 c600 1f2a c000
    0x0000010: 0f2a b600 1e59 c700 06b8 0029 5912 0ab8
    0x0000020: 0027 b800 2259 c700 0557 2b59 4dc0 0015
    0x0000030: 594e b900 2a01 009e 0007 04a7 0004 0399
    0x0000040: 000f 2a2c b600 2059 1209 b800 27b0 2ac0
    0x0000050: 000d b0                                
  Stackmap Table:
    full_frame(@28,{Object[#19],Object[#23]},{Object[#20],Object[#15],Object[#23]})
    same_locals_1_stack_item_frame(@41,Object[#22])
    full_frame(@43,{Object[#19]},{Object[#23]})
    append_frame(@62,Top,Object[#23])
    same_locals_1_stack_item_frame(@63,Integer)
    chop_frame(@78,2)

	at cw$a.a(Unknown Source)
	at bT.createElement(Unknown Source)
	at com.intellij.psi.impl.source.tree.CompositeElement.createPsiNoLock(CompositeElement.java:739)
	at com.intellij.psi.impl.source.tree.CompositeElement.getPsi(CompositeElement.java:726)
	at com.intellij.psi.impl.source.SourceTreeToPsiMap.treeElementToPsi(SourceTreeToPsiMap.java:30)
	at com.intellij.psi.impl.source.tree.SharedImplUtil.getNextSibling(SharedImplUtil.java:60)
	at com.intellij.extapi.psi.ASTDelegatePsiElement.getNextSibling(ASTDelegatePsiElement.java:110)
	at com.intellij.psi.SyntaxTraverser$PsiApi.next(SyntaxTraverser.java:308)
	at com.intellij.psi.SyntaxTraverser$PsiApi.next(SyntaxTraverser.java:264)
	at com.intellij.psi.SyntaxTraverser$ApiEx$1.fun(SyntaxTraverser.java:254)
	at com.intellij.util.containers.JBIterable$2$1.nextImpl(JBIterable.java:125)
	at com.intellij.util.containers.JBIterator.peekNext(JBIterator.java:150)
	at com.intellij.util.containers.JBIterator.hasNext(JBIterator.java:110)
	at com.intellij.util.containers.TreeTraversal$BiOrderIt.nextImpl(TreeTraversal.java:454)
	at com.intellij.util.containers.JBIterator.peekNext(JBIterator.java:150)
	at com.intellij.util.containers.JBIterator.hasNext(JBIterator.java:110)
	at com.intellij.codeInsight.daemon.impl.ChameleonSyntaxHighlightingPass.collectInformationWithProgress(ChameleonSyntaxHighlightingPass.java:91)
	at com.intellij.codeInsight.daemon.impl.ProgressableTextEditorHighlightingPass.doCollectInformation(ProgressableTextEditorHighlightingPass.java:84)
	at com.intellij.codeHighlighting.TextEditorHighlightingPass.collectInformation(TextEditorHighlightingPass.java:69)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$null$1(PassExecutorService.java:430)
	at com.intellij.openapi.application.impl.ApplicationImpl.tryRunReadAction(ApplicationImpl.java:1166)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$doRun$2(PassExecutorService.java:423)
	at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:586)
	at com.intellij.openapi.progress.impl.CoreProgressManager.executeProcessUnderProgress(CoreProgressManager.java:532)
	at com.intellij.openapi.progress.impl.ProgressManagerImpl.executeProcessUnderProgress(ProgressManagerImpl.java:86)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.doRun(PassExecutorService.java:422)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.lambda$run$0(PassExecutorService.java:398)
	at com.intellij.openapi.application.impl.ReadMostlyRWLock.executeByImpatientReader(ReadMostlyRWLock.java:164)
	at com.intellij.openapi.application.impl.ApplicationImpl.executeByImpatientReader(ApplicationImpl.java:218)
	at com.intellij.codeInsight.daemon.impl.PassExecutorService$ScheduledPass.run(PassExecutorService.java:396)
	at com.intellij.concurrency.JobLauncherImpl$VoidForkJoinTask$1.exec(JobLauncherImpl.java:174)
	at java.util.concurrent.ForkJoinTask.doExec(ForkJoinTask.java:289)
	at java.util.concurrent.ForkJoinPool$WorkQueue.runTask(ForkJoinPool.java:1056)
	at java.util.concurrent.ForkJoinPool.runWorker(ForkJoinPool.java:1692)
	at java.util.concurrent.ForkJoinWorkerThread.run(ForkJoinWorkerThread.java:157)
```
## Error context
```
Last Action:       null
Plugin Version:    0.10.3
OS Name:           Windows 10
Java Version:      1.8.0_202-release
Java VM Vendor:    JetBrains s.r.o
Product Name:      IDEA
Full Product Name: IntelliJ IDEA
App Version Name:  IntelliJ IDEA
Is EAP:            false
App Build:         IU-191.7479.19
App Version:       2019.1.3
```

**Comment by karollewandowski** [comment]
Fixed in 0.10.4. Please update the plugin.
